app/Interfaces/CrudRepositoryInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Interfaces;
+
+interface CrudRepositoryInterface
+{
+    public function getAll();
```

Comment:
An interface can also force return values. And docblocks add value to this particular function because I would expect it to return an array of objects, so its nice to know WHICH objects are returned.

Reply:
Added! As discussed the interface has been renamed.